lib/icingadb/icingadb.cpp

```diff
@@ -93,23 +90,10 @@ void IcingaDB::ExceptionHandler(boost::exception_ptr exp)
 		<< "Exception during redis operation: " << DiagnosticInformation(exp);
 }
 
-void IcingaDB::ReconnectTimerHandler()
-{
-	m_WorkQueue.Enqueue([this]() { TryToReconnect(); });
-}
-
-void IcingaDB::TryToReconnect()
+void IcingaDB::OnConnectedHandler()
 {
 	AssertOnWorkQueue();
 
-	if (m_ConfigDumpDone)
```

Comment:
Is this the only such check to remove not to perform the config dump only once?

Reply:
Not sure if I got your question. I removed this one because it's redundant with `if (m_ConfigDumpInProgress || m_ConfigDumpDone)`

Reply:
My question: you restart Redis – do we re-dump?

Reply:
No, we don't. This PR leaves this behavior as is.